assets/sass/components/global/_googlesitekit-entity-header.scss

```diff
@@ -44,21 +49,21 @@
 				top: 180px;
 			}
 
-			@media ( min-width: $bp-desktop ) {
+			@media ( min-width: $width-desktop + 1 + px ) {
 				top: 196px;
 			}
 		}
-
-		@media (min-width: $bp-tablet) {
-			padding-left: $grid-gap-desktop * 1.5;
-			padding-right: $grid-gap-desktop * 1.5;
-		}
 	}
 
-	.googlesitekit-entity-header--has-scrolled {
-		box-shadow: 0 3px 3px rgb(0 0 0 / 10%);
-		transition: box-shadow $t-speed-2;
+	/* stylelint-disable no-descending-specificity */
```

Comment:
Is this necessary? I would think we could merge this block with the `.googlesitekit-entity-header` block above, no? 🤔

Reply:
@aaemnnosttv Actually it was pretty much impossible to have the desired generated CSS since when using the `&` to reference the parent, it references the compiled parent selector. In order to avoid the "hacky" like stylelint disable, I've moved the styles to the related also `assets/sass/components/global/_googlesitekit-dashboard-navigation.scss` where we don't need the stylelint comments.